blueprints/controllers/ikea_e2001_e2002/ikea_e2001_e2002.yaml

```
@@ -315,11 +315,11 @@ variables:
       button_right_long: [hold_3328_0]
       button_right_release: [release_-27903]
       button_up_short: ['on']
-      button_up_long: [move_with_on_off_0_83]
-      button_up_release: [stop]
+      button_up_long: [move_with_on_off_MoveMode.Up_83]
+      button_up_release: [stop_with_on_off]
```

The new events are correct but the change doesn't include the old events for backwards compatibility.
```suggestion
      button_up_long: [move_with_on_off_0_83, move_with_on_off_MoveMode.Up_83]
      button_up_release: [stop, stop_with_on_off]
```